components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -467,15 +471,24 @@ const UserSettingsPage = ({ user }: UserSettingsPageProps) => {
                 }}
               >
                 <div className="flex flex-col gap-3">
-                  <label className="text-2xl font-normal">Delete Account</label>
+                  <label htmlFor="delete-account-button" className="text-2xl font-normal">
+                    Delete Account
+                  </label>
```

Comment:
Refactor (blocking): This label is not necessary, it should be a heading.

```suggestion
                  <h2 htmlFor="delete-account-button" className="text-2xl font-normal">
                    Delete Account
                  </p>
```